examples/pgbouncer/README.md

```diff
@@ -0,0 +1,260 @@
+# Using the Cloud SQL Auth Proxy with PgBouncer
+
+This example demonstrates how to run the Cloud SQL Auth Proxy with PgBouncer on
+Kubernetes. It assumes you have already successfully completed all the steps in
+[Using the Cloud SQL Auth Proxy on Kubernetes][k8s-example].
+
+In this example, you will deploy [PgBouncer][] with the proxy as a sidecar, in
+addition to configuring encryption between the application and PgBouncer.
+
+## Warning
+
+This is an advanced example. Running PgBouncer may be a significant operational
+burden. Nonetheless, running the proxy with PgBouncer may be an appropriate
+choice when running the proxy as a sidecar no longer works effectively at your
+scale.
```

Comment:
I think this section needs to be expanded a bit. Ideally I'd like it to include that:
1. The sidecar is the recommended choice for most users because ... (simpler, less overhead, more secure, lower latency)
2. The service pattern is useful when you are (at _very_ large scale, need a middleware connection pooler, run into SQL Admin API quota problems). 

Bonus points if we can include a relative link to the sidecar example.

Reply:
Done.